server/api/boundaries/findingsDownload.post.ts

```diff
@@ -21,6 +23,9 @@ export default defineEventHandler(async (event) => {
 
   setResponseHeader(event, "Content-Disposition", 'attachment; filename="Findings.txt"');
```

Comment:
might be worthwhile to have this match the frontend where it's `${boundaryname}-findings.xlsx` or something like that as the default value but then have the option to use a provided value as the name